modules/nf-core/ultraplex/main.nf

```diff
@@ -47,6 +47,31 @@ process ULTRAPLEX {
 
     """
     ${ultraplex_command}
+
+    ## rename the matched files to be able to emit only the matched files
+    MATCHES=\$( find . -type f -name '*.fastq.gz' ! -name '*_no_match_*.fastq.gz' )
+    for MATCH in \$MATCHES; do
+        mv \$MATCH \${MATCH/.fastq.gz/_matched.fastq.gz}
```

Comment:
There's no prefix?

Reply:
Not at the moment. It makes files called e.g. ```ultraplex_demux_Sample1_matched.fastq.gz``` based on the barcodes in the input file. The ultraplex_demux could be replaced with a custom prefix, that might be useful

Reply:
Apparently the tool has an `outputprefix` option which changes the "demux" (but not the `ultraplex_`), so I've done that to be the `prefix`.

Reply:
@mahesh-panchal , if you can take a look again please